sphinx/domains/python/_annotations.py

```diff
@@ -265,7 +265,7 @@ def _build_identifier(self, tokens: list[Token]) -> str:
         from itertools import chain, islice
 
         def triplewise(iterable: Iterable[Token]) -> Iterator[tuple[Token, ...]]:
-            # sliding_window('ABCDEFG', 4) --> ABCD BCDE CDEF DEFG
+            # triplewise('ABCDEFG') --> ABC BCD CDE DEF EFG
```

Comment:
Unrelated? (``sliding_window`` is the name for the generic, I named this one ``triplewise`` as it is a concrete version)

Reply:
Ah! I thought it was me who forgot to update the comment! (but my triplewise was using pairwise instead). Will revert.